docs/sources/user_guide/runtime_assurance.rst

```diff
@@ -231,7 +231,7 @@ The RAM publishes a set of markers to visualize the processing done by the Safet
 
 |
 
-This data is published on the **/lll/ram/markers** topic. The :ref:`control panel <control_panel_viz>` can provide you with a an rviz configuration file to visualize this data.
+This data is published on the **/lll/ram/markers** topic. The :ref:`control panel <control_panel_viz>` can provide an rviz configuration file to visualize this data.
 
 .. important::
-  If the **Use Localization** setting is set to false, the robot will stay still at the center of the world, and the raw laserscan data likely won't line up with the Safety Map markers.
+  If the **Use Localization** setting is set to false, make sure to set the :ref:`World Frame <config_sup_world_frame>` parameter to the robot base frame, otherwise, the robot representation will stay still at the center of the world, and the raw laserscan data likely won't line up with the Safety Map markers.
```

Comment:
Why does the raw laserscan data not align with the Safety Map markers?  Isn't the Constraint Map based on the local calculations using the laserscan?

Reply:
Because we introduce an inconsistency between the TF info used by Rviz (which will most likely use localisation), and the internal world->sensor transform the supervisor uses internally. Try it, you'll see what I'm talking about.